linux_os/guide/system/software/integrity/crypto/ssh_client_use_strong_rng/oval/shared.xml

```diff
@@ -0,0 +1,40 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Ensure 'RekeyLimit' is configured with the correct value in /etc/ssh/ssh_config and /etc/ssh/ssh_config.d/*.conf") }}}
+    <criteria comment="ssh client is configured to use strong entropy" operator="AND">
+      <criterion comment="check configuration in /etc/profile.d/cc-ssh-strong-rng.sh" test_ref="test_ssh_client_strong_rng_sh" />
+      <criterion comment="check configuration in /etc/profile.d/cc-ssh-strong-rng.csh" test_ref="test_ssh_client_strong_rng_csh" />
+    </criteria>
+  </definition>
+
+
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check correct entropy configuration in /etc/profile.d/cc-ssh-strong-rng.sh" id="test_ssh_client_strong_rng_sh" version="1">
+    <ind:object object_ref="obj_ssh_client_strong_rng_sh"/>
+    <ind:state state_ref="state_ssh_client_strong_rng" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="obj_ssh_client_strong_rng_sh" version="1">
+    <ind:filepath>/etc/profile.d/cc-ssh-strong-rng.sh</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*export SSH_USE_STRONG_RNG=([\d]+)$</ind:pattern>
+    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check correct entropy configuration in /etc/profile.d/cc-ssh-strong-rng.csh" id="test_ssh_client_strong_rng_csh" version="1">
+    <ind:object object_ref="obj_ssh_client_strong_rng_csh"/>
+    <ind:state state_ref="state_ssh_client_strong_rng" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="obj_ssh_client_strong_rng_csh" version="1">
+    <ind:filepath>/etc/profile.d/cc-ssh-strong-rng.csh</ind:filepath>
```

Comment:
You also need to verify that /etc/profile is set correctly for this configuration. Otherwise, it can be overwritten and the scanner wouldn't know about it at all. That's a bad thing.